db/db_basic_test.cc

```diff
@@ -737,16 +737,21 @@ TEST_F(DBBasicTest, MultiGetSimple) {
 
     std::vector<Slice> keys({"k1", "k2", "k3", "k4", "k5", "no_key"});
 
-    std::vector<std::string> values(20, "Temporary data to be overwritten");
+    std::string tmp_string("Temporary data to be overwritten");
```

Comment:
I am not sure what was the purpose of this initialization. I think it could safely be skipped in PinnableSlice.

Reply:
I think this initialization is merely intended that MultiGet call will overwrite the existing content in the string passed in as part of `values`

Reply:
I cannot think of a bug that would be challenged by this initialization.